server/middlewares/userValidations.js

```diff
@@ -0,0 +1,42 @@
+import Joi from 'joi';
```

Comment:
this imports Joi which is an npm package. This package can be used to validate user inputs to ensure that only the correct format and datatypes are being sent to the model to be saved.